tests/scripts/all.sh

```diff
@@ -1390,6 +1390,7 @@ component_build_module_alt () {
     scripts/config.py unset MBEDTLS_AESNI_C
     scripts/config.py unset MBEDTLS_DEBUG_C
     scripts/config.py unset MBEDTLS_THREADING_PTHREAD
+    scripts/config.py unset MBEDTLS_PK_PARSE_EC_EXTENDED
```

Comment:
Actually, shouldn't we have a `check_config.h` check for that (and the others above)?

Edit: probably not, the limitations are documented in the next commit and a more subtle than what `check_config.h` can handle. OK.